test/gatekeeper.yaml

```diff
@@ -0,0 +1,73 @@
+# in order to connect to cluster after provisioning
```

Comment:
Shouldn't this be part of the platform test suite ?

Reply:
I didn't realise it was already included there, will remove